scripts/mkcloud

```diff
@@ -771,9 +773,54 @@ function testsetup()
     onadmin testsetup
     local ret=$?
     $scp root@$adminip:tempest.log "$artifacts_dir/tempest.log"
+
+    # Register the cloud in Rally and import the results
+    if [ -n "$rally_server" ]; then
+        local title=$(testname)
+        local type=$(testtype)
+
+        $scp root@$adminip:.openrc "$artifacts_dir/.openrc"
+        $scp "$artifacts_dir/tempest.log" rally@$rally_server:/tmp/tempest.$$.log
```

Comment:
Wouldn't it be better to use `mktemp -d` (through ssh) instead of the `$$` bits?